packages/synapse-react-client/src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -208,71 +213,85 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
 
   return (
     <SynapseErrorBoundary>
-      <div className="bootstrap-4-backport EntityFinder">
-        <div className="EntityFinder__Search" data-active={searchActive}>
+      <div className="EntityFinder">
+        <Box
+          display="flex"
+          justifyContent="flex-end"
+          alignItems="center"
+          pb={1}
+        >
           <>
-            {searchActive ? (
+            {!searchActive ? (
               <Button
-                variant="gray-primary-500"
-                onClick={() => {
-                  setSearchActive(false)
-                  setSearchInput('')
-                  setSearchTerms(undefined)
-                }}
-              >
-                <Arrow arrowDirection="left" style={{ height: '16px' }} />
-                Back to Browse
-              </Button>
-            ) : (
-              <Button
-                variant="sds-primary"
+                variant="contained"
+                color="primary"
                 className="EntityFinder__Search__SearchButton"
                 onClick={() => {
                   setSearchActive(true)
-                  searchInputRef.current!.focus()
```

Comment:
Use `autoFocus` to move focus to Search field, rather than setting ref on clicking the search button

Reply:
To apply the transition, the TextField needs to be hidden initially, but then the input focus needs to shift once the EntityFinder is in search mode. Using autoFocus won't work in this case, since the focus would only be applied when the component was initially rendered. Revert to using the ref